src/components/picker/index.tsx

```diff
@@ -208,21 +203,21 @@ const Picker = React.forwardRef((props: PickerProps, ref) => {
         onCancel: cancelSelect
       };
 
-      return renderCustomModal(modalProps);
+      return componentProps?.customModal && componentProps.customModal?.(modalProps);
     }
   };
 
   const expandableModalContent = useMemo(() => {
-    const useItems = useWheelPicker || propItems;
+    const useItems = type.wheelPicker || propItems;
+    const listTopBarProps = componentProps.headerProps;
```

Comment:
Why assign to another const if you use it once? Also, it is difficult to keep track if your code is not consistent...